docs/chaos-engineering/User Guides/connect-chaos-infrastructures.md

```diff
@@ -0,0 +1,51 @@
+
+![New Environment](./static/connect-chaos-infrastructures/new-environment.png)
+
+## Add Chaos Infrastructure
+Now we can add a chaos infrastructure to our environment. Select **New Chaos Infrastructure**.
+
+You can either setup a chaos infrastructure on existing infrastructures that uses a Harness cluster connector i.e. Harness Kubernetes delegate or install the chaos infrastructure in a new infrastructure.
+
+To use an existing Harness Kubernetes delegate to install your chaos infrastructure, select **On Existing Infrastructures** in the modal. You can use any delegate under the Project, Organization, and Account scope.
+
+![Chaos Infrastructure in Existing Infra](./static/connect-chaos-infrastructures/chaos-infrastructure-in-existing-infra.png)
+
+:::info
+1. A Harness Kubernetes delegate with cluster-wide read/write access can setup both cluster-scoped and namespaced-scoped chaos infrastructures.
+2. A Harness Kubernetes delegate with cluster-wide read access **can't** be used to setup a chaos infrastructures.
+3. A Harness Kubernetes delegate with specific namespace access can only be used to setup a namespace-scoped chaos infrastructure in that specific Kubernetes namespace.
+:::
+
+To install chaos infrastructure on a new infrastructure, select **On New Infrastructures**. Then, select **Continue**.
+Provide a name for the infrastructure. Optionally, you can also provide a description and tags. Then, select **Next**. 
+
+Under the installation mode, choose either **Cluster Wide** or **Namespace Mode** for installing the chaos infrastructure. By default the installation will take place in the `litmus` namespace and uses `litmus` service account, which can be configured under the K8s Cluster Details. Optionally, you can also specify node selectors and K8s tolerations for chaos infrastructure deployment.
+
+![Configure Chaos Infrastructure](./static/connect-chaos-infrastructures/configure-chaos-infrastructure.png)
+
+Then, select **Next**. In the final step, if you're deploying to an existing infrastructure with Harness Kubernetes Delegate, then Harness will install the chaos infrastructure on your behalf. Otherwise, if you're setting up the chaos infrastructure on a new infrastructure, then you will need to execute the given commands and/or download and apply the installation manifest.
+
+![k8s Setup Infrastructure](./static/connect-chaos-infrastructures/k8s-setup-infrastructure.png)
+
+Finally, select **Done**.
+
+## Validating Chaos Infrastructure Installation
+Upon applying the manifest, it will take a while to setup all the chaos infrastructure resources. After a while, the chaos infrastructure's connection state should reflect as `CONNECTED`.
+
+If the chaos infrastructure remains in a `NOT CONNECTED` state, follow the suggestions on the **Troubleshooting** tab to resolve it.
+
+![Infrastructure State](./static/connect-chaos-infrastructures/infrastructure-state.png)
```

Comment:
This screen has been updated in the [stress](https://stress.harness.io/) can you please update it from there